[2022-08-17 19:50:19,921] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Deepface.validate manual__2022-08-17T19:50:14.515165+00:00 [queued]>
[2022-08-17 19:50:20,092] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Deepface.validate manual__2022-08-17T19:50:14.515165+00:00 [queued]>
[2022-08-17 19:50:20,093] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-17 19:50:20,093] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-08-17 19:50:20,093] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-17 19:50:20,109] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): validate> on 2022-08-17 19:50:14.515165+00:00
[2022-08-17 19:50:20,120] {standard_task_runner.py:52} INFO - Started process 3019 to run task
[2022-08-17 19:50:20,123] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Deepface', 'validate', 'manual__2022-08-17T19:50:14.515165+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/deepface.py', '--cfg-path', '/tmp/tmpu_7vbi9t', '--error-file', '/tmp/tmphsuft5uq']
[2022-08-17 19:50:20,124] {standard_task_runner.py:80} INFO - Job 57: Subtask validate
[2022-08-17 19:50:20,186] {task_command.py:370} INFO - Running <TaskInstance: Deepface.validate manual__2022-08-17T19:50:14.515165+00:00 [running]> on host d254c144b912
[2022-08-17 19:50:20,277] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Deepface
AIRFLOW_CTX_TASK_ID=validate
AIRFLOW_CTX_EXECUTION_DATE=2022-08-17T19:50:14.515165+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-17T19:50:14.515165+00:00
[2022-08-17 19:50:27,636] {logging_mixin.py:115} INFO - 1/1 [==============================] - ETA: 0s
[2022-08-17 19:50:27,637] {logging_mixin.py:115} INFO - 1/1 [==============================] - 1s 770ms/step
[2022-08-17 19:50:28,117] {logging_mixin.py:115} INFO - 1/1 [==============================] - ETA: 0s
[2022-08-17 19:50:28,117] {logging_mixin.py:115} INFO - 1/1 [==============================] - 0s 358ms/step
[2022-08-17 19:50:31,134] {logging_mixin.py:115} INFO - Email sent successfully!
[2022-08-17 19:50:31,134] {python.py:173} INFO - Done. Returned value was: None
[2022-08-17 19:50:31,153] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Deepface, task_id=validate, execution_date=20220817T195014, start_date=20220817T195019, end_date=20220817T195031
[2022-08-17 19:50:31,293] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-17 19:50:31,393] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
